tests/unit/test_main.py

```
@@ -5,235 +5,209 @@
+            mock_response.text = "<html><body><h1>Test</h1></body></html>"
+            mock_response.raise_for_status.return_value = None
+            mock_request.return_value = mock_response
+
+            results = converter.convert_url_list(
+                urls=urls,
+                output_dir=temp_dir,
+                output_format="markdown",
+                save_chunks=False,
+            )
+
+            assert len(results) == 2
```

Ruff (0.11.9)

Magic value used in comparison, consider replacing `2` with a constant variable

(PLR2004)